dsl/src/main/scala/com/crobox/clickhouse/dsl/misc/DSLImprovements.scala

```diff
@@ -128,7 +128,7 @@ object DSLImprovements {
         //            case _                    =>
         //          }
         case _: TableFromQuery[_] => return query.andConstraint(condition)
-        case _ =>
+        case null =>
```

Comment:
Why is this now matching on `null` instead of `_`?

Reply:
When compiled I have got the warning saying that the only case in which this would be captured, it is if the matched variable was null, so I replaced it, can replace back and see how it compiles in CI/CD